src/components/shared/UpperSection.tsx

```diff
@@ -8,29 +9,82 @@ interface upperSectionProps {
 
 //UpperSection contains description, code and buttons
 export default function UpperSection(props:upperSectionProps): JSX.Element {
+  // As of right now, we have description being a string. Should it be an array?
   let description;
   let buttons;
-  const pages = ['/','/second','/third'];
+  //Stores types of pages that are moved through in nextPage
+  const pages = ['/', '/EasyIf','/HungerIfElse', '/ObstacleIfElse', '/IfElif', '/Nested'];
   const location = useLocation();
+  // Finds path name of file and also serves for nextPage
   const currentPage = location.pathname;
+  // Automatically finds nextPage to be used for ChoiceButton
   const nextPage = pages.indexOf(currentPage) === pages.length-1 ? '' : pages[pages.indexOf(currentPage)+1];
 
-  //Parameters for nextPage are (addbattery,addspeed,addstrength,addhunger)
+  // Parameters for nextPage are (addbattery,addspeed,addstrength,addhunger)
+  // Should I assume that the above parameters should be added to a different variable?
+  // Also, have we decided on specific values for buttons?
   switch (currentPage) {
     case '/':
-      description = 'landing text a;sldfk jasl;d fjk asl ;dfkj asl;dfj as; dlfkja  s;dl  fk   ja s; dlf kj ljk asd f;l k ajsd f;la  sjkdfa ; ';
+      // Do we need a code component as well?
```

Comment:
Code component is being implemented in another issue so we can remove this comment